### PR TITLE
DP-10440: FooterSlim Component

### PR DESCRIPTION
**Pull request by mark-casias**
## Description
Ports slim footer component from EEC to Mayflower React.

## Related Issue / Ticket
- [JIRA issue](https://jira.mass.gov/browse/DP-10440)

## Steps to Test
1. `npm run start`
1. View at http://localhost:6006/?selectedKind=organisms&selectedStory=FooterSlim&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

## Screenshots
image

### CODE REVIEW

File: react/src/components/organisms/FooterSlim/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import './style.css';
+
+export default () => (
+  <footer className="ma__footer_slim" id="footer">
+    <div className="ma__footer_slim--container ma__container">
+      <SiteLogo />
+      <div className="ma__footer_slim--container-inner">
+        <section className="ma__footer_slim--info">
+          <h3>Massachusetts Executive Office of Eductation (EDU)</h3>
```
Comment (avrilpearl) [inline react/src/components/organisms/FooterSlim/index.js]: @mark-casias --> all of these need to be passable props

File: react/src/components/organisms/FooterSlim/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          <p>
+            The Department of Early Education and Care&apos;s mission is to
```
Comment (avrilpearl) [inline react/src/components/organisms/FooterSlim/index.js]: should be passable prop

File: react/src/components/organisms/FooterSlim/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            support the healthy growth and development of all children by
+            providing high quality programs and resources for families
+          </p>
+          <p className="ma__footer_slim--copyright">
+            &copy; 2018 Commonwealth of Massachusetts
+          </p>
+        </section>
+        <section className="ma__footer_slim--details">
+          <div className="ma__footer_slim--links">
```
Comment (avrilpearl) [inline react/src/components/organisms/FooterSlim/index.js]: make array of passable links + title

File: react/src/components/organisms/FooterSlim/style.scss [hunk 4]
```diff
@@ -0,0 +1,105 @@
+@import "global";
```
Comment (avrilpearl) [inline react/src/components/organisms/FooterSlim/style.scss:1]: move this out of here and into shared assets level as it is a new component

File: react/src/components/organisms/FooterSlim/FooterSlim.stories.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs, object } from '@storybook/addon-knobs/react';
+
+import FooterSlim from './index';
+
+storiesOf('organisms', module)
+  .addDecorator(withKnobs)
+  .add('FooterSlim', withInfo('{ FooterReadme }')(() => {
+    return(<FooterSlim />);
```
Comment (avrilpearl) [inline react/src/components/organisms/FooterSlim/FooterSlim.stories.js]: once props are passable, add knobs to story

**Review by smurrayatwork — APPROVED** [review]

**Pull request merged**